__tests__/api.test.js

```diff
@@ -311,4 +311,20 @@ describe('DELETE /api/comments/:comment_id', () => {
     })
 })
 
+describe('GET /api/users', () => {
+    test('GET: 200 should send an array of users with the properties of username, name and avatar_url to the client', () => {
+        return request(app).get("/api/users").expect(200)
+       .then((response) => {
+           const {users}= response.body
+           expect(users).toBeInstanceOf(Array)
+           users.forEach(user => {
```

Comment:
Remember to check the length of the array before iterating through or you could get a false positive